Tests/TuistSigningTests/SigningInstallerTests.swift

```diff
@@ -0,0 +1,101 @@
+import Foundation
+import TSCBasic
+import TuistSupport
+import XCTest
+@testable import TuistCoreTesting
+@testable import TuistSigning
+@testable import TuistSigningTesting
+@testable import TuistSupportTesting
+
+final class SigningInstallerTests: TuistUnitTestCase {
+    var subject: SigningInstalling!
+    var securityController: MockSecurityController!
+
+    override func setUp() {
+        super.setUp()
+        try? ProcessEnv.setVar("TUIST_VERBOSE", value: "true")
```

Comment:
Does this variable remain after the test has completed? If so, we need to figure out a different way to inject the "verbose" value. Otherwise, having a global variable might cause other tests to fail.

Reply:
You should not need to set this, by default in tests the logging is set to trace and you just need to use `XCTAssertPrinterOutputContains`, `XCTAssertPrinterErrorContains` or `XCTAssertPrinterContains` which allows you to also specify a specific log level if you also want to check that